src/platform/web.rs

```diff
@@ -17,10 +17,25 @@ pub trait WindowExtWebSys {
 }
 
 pub trait WindowBuilderExtWebSys {
+    /// Enable scrolling of the web page the canvas is in when the canvas is focused.
+    ///
+    /// Scrolling is disabled by default because the scroll input on many mobile devices
+    /// is the same as click and dragging which is a very common input method for many applications.
+    ///
+    /// So only call this method if you know that you will never need to handle mouse wheel inputs
+    /// or click and dragging.
+    fn enable_web_scroll(self) -> Self;
```

Comment:
Suggestion: `enable_scroll`?

Reply:
I think the web there is important, the user needs to know this is a web specific hack.

Reply:
Despite the method being on a trait called WindowBuilderExtWebSys when the method is called there is no indication which trait the method comes from.
As such I think its better to make it explicit that this is a web specific method.
I think `enable_web_scroll` does this well enough.
But now I'm thinking we should go a step further and name it `enable_web_page_scroll` to help avoid the misunderstanding that this affects the canvas's ability to receive scroll events.